source/usage-examples.txt

```diff
@@ -32,3 +15,52 @@ Usage Examples
    /usage-examples/command
    /usage-examples/changeStream
    /usage-examples/bulkWrite
+
+Overview
+--------
+
+Usage examples provide convenient starting points for popular MongoDB
+operations. Each example provides:
+
+- an explanation of the operation in the example showing the
+  purpose and a sample use case for the method
+
+- an explanation of how to use the operation, including parameters,
+  return values, and common exceptions you might encounter
+
+- a full Node.js program that you can copy and paste to run the example
+  in your own environment
+
+How to Use the Usage Examples
+-----------------------------
+
+These examples use the :atlas:`MongoDB Atlas sample data <sample-data>`
+database. You can use this sample data on the free tier
+of MongoDB Atlas by following the :atlas:`Get Started with Atlas
+<getting-started/#atlas-getting-started>` guide or you can
+:manual:`import the sample dataset into a local MongoDB instance
+</import/>`.
+
+Once you have imported the dataset, you can copy and paste a usage
+example into your development environment of choice. You can follow the
+:doc:`quick start guide </quick-start>` to learn more about getting
+started with Node.js, npm, and the Node.js driver. Once you've copied
+a usage example, you'll have to edit one line to get the example running
+with your instance of MongoDB:
+
+.. code-block:: javascript
+
+   // Replace the following with your MongoDB deployment's connection string.
+   const uri =
+      "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+You can use the :guides:`Atlas Connectivity Guide
+</cloud/connectionstring/>` to enable connectivity to your instance of
+Atlas and find the :manual:`connection string
+</reference/connection-string/>` to replace the ``uri`` variable in the
+usage example. If your instance uses :manual:`SCRAM authentication
+</core/security-scram/>`, you can replace ``<user>`` with your username,
+``<password>`` with your password, and ``<cluster-url>`` with the IP
+address or URL of your instance. Consult the :doc:`authentication guide
```

Comment:
I see in the txt file it says to consult the `authentication guide` but when it's rendered it just says consult the "Authentication"

<img width="951" alt="Screen Shot 2020-02-24 at 11 53 44 AM" src="https://user-images.githubusercontent.com/52428905/75173196-84eb9380-56fc-11ea-98bd-a2500c917c18.png">

Reply:
Good eye! I'll ask platform team about this -- seems like it should take the supplied text rather than the title of the doc based on how the other directives work, but maybe they decided it should work otherwise. Will make appropriate fix before merging.